src/install/elasticsearch/whatsNext.mdx

```diff
@@ -260,7 +260,7 @@ The Elasticsearch integration collects both Metrics and Inventory information. I
       </td>
 
       <td>
-        A way to further specify which cluster we are gathering data for, example:
+        A way to further specify which cluster we are gathering data, for example:
```

Comment:
Alternate: You can specify the cluster from which you are gathering data, for example, 'staging'.

Reply:
Done!